File: 387-first-unique-character-in-a-string/first-unique-character-in-a-string.c
int firstUniqChar(char* s) {
        int i, j;
    int n = strlen(s);

    for (i = 0; i < n; i++) {
        int c = 0;
        for (j = 0; j < n; j++) {
            if (i != j && s[i] == s[j]) {
                c = 1;
                break;
            }
        }
        if (c == 0) {
            return i;
        }
    }
    return -1;
}


        



    

File: 387-first-unique-character-in-a-string/first-unique-character-in-a-string.java
class Solution {
    public int firstUniqChar(String s) {
        int len = s.length();
        for (int i = 0; i < len; i++) {
            int c = 0;
            for (int j = 0; j < len; j++) 
            {
                if ( i != j && s.charAt(i) == s.charAt(j)) {
                    c = 1;
                    break;  
                }
            }
            if (c == 0) {
                return i;
            }   
        }
        return -1; 
    }
}